contacts/migrations/0001_initial.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.13 on 2018-07-24 23:59
+from __future__ import unicode_literals
+
+import django.contrib.gis.db.models.fields
+from django.db import migrations, models
+import phonenumber_field.modelfields
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Contact',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('email_address', models.EmailField(blank=True, max_length=255, null=True)),
+                ('external_id', models.CharField(blank=True, max_length=128, null=True, unique=True)),
+                ('first_name', models.CharField(blank=True, max_length=255, null=True)),
+                ('last_name', models.CharField(blank=True, max_length=255, null=True)),
+                ('point', django.contrib.gis.db.models.fields.PointField(blank=True, null=True, srid=4326)),
```

Comment:
Do we see any value to storing zipcode? It's the most basic location information we store on pretty much all constituents and we sometimes use it to target. Although perhaps point covers us here?

Reply:
Rather than add all the fields that might be useful in the future, I focused on adding the fields that I know we need now. So I suggest we can just iterate in the future assuming we like this pattern in general.

Reply:
Cool, agreed.